packaging/helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -120,6 +120,12 @@ kanikoExecutor:
     repository: strimzi
     name: kaniko-executor
     tag: latest
+mavenImage:
+  image:
+    registry: quay.io
+    repository: strimzi
+    name: maven-builder
+    tag: latest
```

Comment:
Should these be also added the the README.md of HelmChart?

Reply:
https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/DO_NOT_EDIT.md